core/src/raw/chrono_util.rs

```diff
@@ -62,3 +62,10 @@ pub fn parse_datetime_from_from_timestamp(s: i64) -> Result<DateTime<Utc>> {
 
     Ok(st.into())
 }
+
+/// format datetime to rfc1123
+///
+/// for example: `Sun, 06 Nov 1994 08:49:37 GMT`
+pub fn to_rfc_1123(s: DateTime<Utc>) -> String {
```

Comment:
I suggest to use `format_datetime_into_http_date` as API name.

Besides, how about adding a test case for this?